### PR TITLE
We need to mark the status property as nullable for swift support

### PR DESCRIPTION
**Pull request by jdizoglio**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jdizoglio has signed the CLA. Thanks for the pull request!

**Review by boxtcarpel — APPROVED** [review]

File: BoxContentSDK/BoxContentSDK/Models/BOXItem.h [hunk 1]
```diff
@@ -148,7 +148,7 @@
  *  Whether this item is deleted or not.
  *  Values include "active", "trashed" if the file has been moved to the trash, and "deleted" if the file has been permanently deleted
  */
-@property (nonatomic, readwrite, strong) NSString *status;
+@property (nonatomic, readwrite, copy, nullable) NSString *status;
```
Comment (boxtcarpel) [inline BoxContentSDK/BoxContentSDK/Models/BOXItem.h:151]: Should we update the other properties as well?

**Comment by thuyBox** [comment]
+2

File: BoxContentSDK/BoxContentSDK/Models/BOXItem.h (hunk 1 above)
Comment (jdizoglio) [reply BoxContentSDK/BoxContentSDK/Models/BOXItem.h:151]: Yes, I have a ticket open to review the entire class. I will commit to unblock the upstream caller.

**Pull request merged**